src/models/Factor.js

```diff
@@ -210,6 +210,9 @@ function (Okta, Q, factorUtil, Util, Errors, BaseLoginModel) {
       this.appState = attributes.appState;
       // set the initial value for remember device.
       attributes.rememberDevice = factorUtil.getRememberDeviceValue(this.appState);
+
+      // Add vendorname for custom totp enroll
```

Comment:
should we add `vendorName` to the [local props](https://github.com/okta/okta-signin-widget/blob/53c3bb033bf8fe760a1f8cb6c0e15cc274aef9af/src/models/Factor.js#L87)?

Reply:
It is already in props.